inputs/abutton.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+    <ui-icon-button
+    buttonType="button"
+    color="default"
+    type="secondary"
+    :ariaLabel="args.tooltip"
+    :loading="loading"
+    :tooltip="args.tooltip"
+    v-if="isShown"
+    @click="$emit('abuttonclick')">
+    <ui-icon>{{ args.icon || 'check_circle' }}</ui-icon>
```

Comment:
If I'm not mistaken, we can send an `icon` as prop to `ui-icon-button` component. Then, we wouldn't need to insert an `ui-icon` in here. Here's the example:
```vue
<template>
  <ui-icon-button
    buttonType="button"
    color="default"
    type="secondary"
    :ariaLabel="args.tooltip"
    :icon="args.icon || 'check_circle'"
    :loading="loading"
    :tooltip="args.tooltip"
    v-if="isShown"
    @click="$emit('abuttonclick')"
  />
</template>
```

Reply:
👍